Crypto/PubKey/RSA/PSS.hs

```diff
@@ -161,30 +163,37 @@ verifyDigest :: HashAlgorithm hash
              -> ByteString  -- ^ Signature
              -> Bool
 verifyDigest params pk digest s
-    | public_size pk /= B.length s        = False
+    | B.length s /= k                     = False
+    | B.any (/= 0) pre                    = False
```

Comment:
This form `B.any (/= 0) bytes` is IMHO cleaner than the below `not (B.all (== 0) bytes)` used in the `ps0` case.  Perhaps change that one to match?

Reply:
Done with 1d5947f.